src/comb/template.clj

```diff
@@ -18,23 +19,33 @@
         ")?"
         "(.*)\\z")))
 
-(defn emit-string [s]
-  (print "(print " (pr-str s) ")"))
+(defn emit-string
+  ([s]
+     (print "(print " (pr-str s) ")"))
+  ([s emit-next-line-break?]
+     (if emit-next-line-break?
```

Comment:
I don't think ERB supports `<%=-`, or rather, it evaluates to treating the `-` as a subtraction. `<%-=` doesn't work either, though I don't see why it shouldn't. In Comb we might as well support it, and the regex above should do that fine.